capstone/src/main/resources/static/index.html

```diff
@@ -7,6 +7,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
     <script src="script.js"></script>
     <script src="user.js"></script>
+    <script src="event.js"></script>
```

Comment:
glad you moved many functions out of script.js. I think we should just use script.js to hold shared functions.